benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -17,6 +17,9 @@
 
 #include "ppx/graphics_util.h"
 
+// Use this to disable sphere rendering completely, including loading
+//#define DISABLE_SPHERES
```

Comment:
Question: why isn't this a flag knob? Is it because launching XR app with a flag is not that easy?

Reply:
The generating of the VB is at loading time which is taking quite a long time. Currently on XR, the imgui UI cannot be controlled by either controller or mouse, so every time i want to change a knob value, i have to modify the code and restart the app. This is just a quick hackish way to disable the loading of the VB completely for faster iteration to test the bandwidth.

Reply:
Sure, that I get. But why isn't a KnobFlag usable instead?
Value cannot change at runtime, so you can do a `if(disableSphereKnob.GetValue())` around the VB init code no?

Reply:
good point, fixed